openupgrade_scripts/scripts/sale_stock/15.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,24 @@
+from openupgradelib import openupgrade
+
+
+def _rename_field_product_packaging_to_product_packaging_id(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        ALTER TABLE sale_order_line
+        ADD COLUMN IF NOT EXISTS product_packaging_id integer
```

Comment:
As ``sale`` as been updated at this step, the new field ``product_packaging_id`` has been created by the ORM, don't you think ?

Reply:
@legalsylvain Your idea is excellent, however as you stated in the Sale module, it will be migrated first. Because 'product packaging id' has already been generated in this scenario, renaming will result in an error.

Reply:
@legalsylvain This is only a guarantee. I don't think that will be an issue, you may reconsider.